plugins.xml

```diff
@@ -14021,4 +14021,35 @@ the registration of DOIs with mEDRA.</p>]]></description>
 			<description locale="es_ES">Nueva funcionalidad: Actualización de datos preservados en la Red Cariniana. El envío de datos para preservación ahora envía solo el XML actualizado después del primer envío. Las actualizaciones automáticas a través de tareas programadas monitorean diferencias y envían correos electrónicos de actualización a la Red Cariniana. Las tareas programadas se activan automáticamente por el plugin Acron.</description>
 		</release>
 	</plugin>
+	<plugin category="blocks" product="navigation">
+		<name locale="en">Navigation Block Plugin</name>
+		<name locale="en_US">Navigation Block Plugin</name>
+		<homepage>https://github.com/pkp/tutorialExample</homepage>
```

Comment:
Hi @mpbraendle, should this be updated to https://github.com/mpbraendle/navigation/?